datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/openmetrics.yaml

```diff
@@ -223,6 +223,12 @@
   value:
     example: true
     type: boolean
+- name: ignore_connection_errors
```

Comment:
You should run `ddev validate config all -s` to apply this config option to all OpenMetricsV2-based checks.